### DISCUSSION TITLE
Why is 403 HTTP response flagged as an error event?

### DISCUSSION THREAD

**Discussion by krnekhelesh** [getsentry/support#48] [Support] [answered]
We deployed Sentry recently to our production servers and noticed a few issues being flagged. On a closer look, we noticed that these were 403 errors where the user tried accessing a protected resource without actually signing in to our client apps. Isn't 403 a client app error? If so, why is it being flagged as an issue as it gives the impression that something is wrong with the server.

**Comment by davestewart** [comment, marked as answer]
It seems that you can add a `beforeSend()` callback to your config, returning `null` to prevent this:

- https://docs.sentry.io/platforms/javascript/guides/remix/configuration/filtering/#filtering-error-events

**Reply by souredoutlook** [reply]
Hi @krnekhelesh  - @davestewart 's comment about filtering this out using beforeSend is a good approach.

Depending on your framework and how it's configured it's hard to comment on whether this is a mistake, for example, it's possible to configure our Node SDK for Express to report only on specific status codes like so:

```
app.use(
  Sentry.Handlers.errorHandler({
    shouldHandleError(error) {
      // Capture all 404 and 500 errors
      if (error.status === 404 || error.status === 500) {
        return true;
      }
      return false;
    },
  })
);
```
